x-pack/test/functional_endpoint/apps/endpoint/resolver.ts

```diff
@@ -0,0 +1,266 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function({ getPageObjects, getService }: FtrProviderContext) {
+  const pageObjects = getPageObjects(['common', 'timePicker', 'endpointAlerts']);
+  const testSubjects = getService('testSubjects');
+  const esArchiver = getService('esArchiver');
+  const retry = getService('retry');
+
+  describe('Endpoint Alert Resolver', function() {
+    this.tags(['ciGroup7']);
+    before(async () => {
+      const fromTime = 'Sep 22, 2019 @ 20:31:44.000';
+      const toTime = 'Now';
+      await esArchiver.load('endpoint/resolver_tree/api_feature');
+      await pageObjects.common.navigateToUrlWithBrowserHistory('endpoint', '/alerts');
+      await retry.try(async function() {
+        await testSubjects.existOrFail('superDatePickerShowDatesButton');
+      });
```

Comment:
`retry` service should be used when there is a flakiness you can't  avoid.
it will retry operations for up to 40 seconds, it is probably too much for loading the page.

you can use `await testSubjects.existOrFail('superDatePickerShowDatesButton', { timeout: 20000 });`